home/templates/home/home_page.html

```diff
@@ -24,18 +24,19 @@
       </div>
     </header>
 
+    {% for c in categories %}
 
     <!-- Portfolio Grid Section -->
-    <section id="plan">
+    <section {% if forloop.counter|divisibleby:2 %}class=success{% else %}odd{% endif %} id={{c.name}}>
```

Comment:
Just a suggestion but django templating's cycle tag might be more concise here:
https://docs.djangoproject.com/en/1.10/ref/templates/builtins/#cycle